AlinaFirsova/Homework 1/Homework_1_Firsova.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE HTML>
+<html>
+ <head>
+  <meta charset="utf-8">
+  <title>Cubes</title>
+<style type="text/css">
+ @import"style.css"

```

Comment:
@AlinaFirsova Please look through 'link' tag overview 
https://www.w3schools.com/tags/tag_link.asp
If you have external styles you need to use link instead of style